KachnaOnline.Business/Mappings/EventsMappings.cs

```diff
@@ -12,11 +15,33 @@ public class EventMappings : Profile
     {
         public EventMappings()
         {
-            this.CreateMap<EventEntity, Event>().ReverseMap();
+            this.CreateMap<EventEntity, Event>()
+                .ForMember(e => e.LinkedPlannedStateIds, options =>
+                    options.MapFrom(e => e.LinkedPlannedStates.Select(state => state.Id).ToList()));
+            this.CreateMap<Event, EventEntity>();
+
+            this.CreateMap<EventEntity, EventWithLinkedStates>()
+                .ForMember(e => e.LinkedStates, options =>
+                    options.MapFrom(e => e.LinkedPlannedStates))
+                .ForMember(e => e.LinkedPlannedStateIds, options =>
+                    options.MapFrom(e => e.LinkedPlannedStates.Select(state => state.Id).ToList()));
+
             this.CreateMap<EventEntity, NewEvent>().ReverseMap();
```

Comment:
Why do you need a reverse mapping for this?

Reply:
Probably do not need as of right now, because modifying and planning events have their own models and DTOs, but at the time of writing these mappings, I did not know what might be needed in the future and what will not be.

I would not remove these for the time being, however. I do not see a problem with having maybe unused reverse mappings in case it is needed some day.

Reply:
Never mind, I have gone through them and removed unnecessary mappings, where it was possible.